spark/src/main/antlr/SqlBaseParser.g4

```diff
@@ -989,7 +989,7 @@ primaryExpression
     | CASE whenClause+ (ELSE elseExpression=expression)? END                                   #searchedCase
     | CASE value=expression whenClause+ (ELSE elseExpression=expression)? END                  #simpleCase
     | name=(CAST | TRY_CAST) LEFT_PAREN expression AS dataType RIGHT_PAREN                     #cast
-    | primaryExpression collateClause                                                          #collate
+    | primaryExpression collateClause                                                      #collate
```

Comment:
nit: space alignment

Reply:
This is coming from automated step. We need to change in the original PR of opensearch-spark.